lib/postgrex/type_module.ex

```diff
@@ -382,11 +382,12 @@ defmodule Postgrex.TypeModule do
   defp decode_row_dispatch(extension, :super_binary, rest, acc, rem, full, rows) do
     [clause] =
       quote do
-        [{unquote(extension), sub_oids, sub_types} | types] ->
+        [{{unquote(extension), sub_oids, sub_types}, mod} | types] ->
```

Comment:
Can we avoid nesting tuples if possible?

Reply:
I can do it if I flatten during parsing the row description method like this: https://github.com/elixir-ecto/postgrex/pull/684/commits/7057e83fe6fcf5a7ad4e68f4348b1f78e8bb3da9#diff-0da854f0c1cda9486d776c72ecda6a2e595a7667b72688669bbd80d6b80f0f96R1768